src/redux/books/books.js

```diff
@@ -1,7 +1,23 @@
 import { createSlice } from '@reduxjs/toolkit';
 
 const initialState = {
-  books: [],
+  books: [
+    {
+      title: 'The Lord of the Rings',
+      author: 'J.R.R Tolkien',
+      id: '29837928734',
+    },
+    {
+      title: 'The Call of Cthulhu',
+      author: 'H.P Lovecraft',
+      id: 'uyed7273rrd2',
+    },
+    {
+      title: 'Brave New World',
+      author: 'Aldous Huxley',
+      id: 'oijvow98934h7fe66',
+    },
+  ],
```

Comment:
- [ ] Everything is working fine book is added properly and it is removed properly but when I refresh the page then the added book is removed and only hardcopy remains there kindly fix this problem as well.